apex/contrib/layer_norm/layer_norm.py

```diff
@@ -7,47 +7,56 @@
 
 class FastLayerNormFN(torch.autograd.Function):
     @staticmethod
-    def forward(ctx, x, gamma, beta, epsilon):
+    def forward(ctx, x, gamma, beta, epsilon, use_1p=False):
         x = x.contiguous()
         gamma = gamma.contiguous()
         beta = beta.contiguous()
         hidden_size = gamma.numel()
         xmat = x.view((-1, hidden_size))
-        ymat, mu, rsigma = fast_layer_norm.ln_fwd(xmat, gamma, beta, epsilon)
+        if use_1p:
+            one_plus = 1.0
+        else:
+            one_plus = 0.0
+        ymat, mu, rsigma = fast_layer_norm.ln_fwd(xmat, gamma, beta, epsilon, one_plus)
```

Comment:
I would suggest this for simplicity (assumes use_1p is a bool)
```suggestion
        ymat, mu, rsigma = fast_layer_norm.ln_fwd(xmat, gamma, beta, epsilon, float(use_1p))
```
(similarly in other places)